Documentation/upgrades/upgrade_3_2.md

```diff
@@ -0,0 +1,182 @@
+## Upgrade etcd from 3.1 to 3.2
+
+In the general case, upgrading from etcd 3.1 to 3.2 can be a zero-downtime, rolling upgrade:
+ - one by one, stop the etcd v3.1 processes and replace them with etcd v3.2 processes
+ - after running all v3.2 processes, new features in v3.2 are available to the cluster
+
+Before [starting an upgrade](#upgrade-procedure), read through the rest of this guide to prepare.
+
+### Client upgrade checklists
```

Comment:
Yes. Just v3.1 client's`TimeToLive` call to v3.2 server will return -1 for non-existing lease ID.